packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -27,53 +67,22 @@ $ds-c-inset-border-width: $spacer-half;
       margin-left: $spacer-1;
     }
   }
+  // Style radio and checkboxes when errored
+  // &.ds-c-choice--error + label::before {
+  //   border-color: $color-error;
+  // }
```

Comment:
commented out the error styling so child design systems could use if they wanted to